documentation/staking-governance.md

```diff
@@ -13,37 +13,53 @@ This design with selectable locking duration helps to align token holders with t
-- In case a user pauses their locking period countdown, they receives a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200%.
+- Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides 1 vpLSK.
+- In case a user pauses their locking period countdown, they receive a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200% by pausing a locking position with the maximum locking period of 2 years.
+
+### Voting Power for a Concrete Proposal
+
+At the time a proposal is created, a snapshot of the delegated voting power is taken and the voting result is based on this snapshot. Consequently, neither any changes in the voting power nor any delegations made after the proposal creation have any influence on the voting result.
 
 ## Staking Rewards
 
 Users receive rewards for their locking tokens as follows: 
 
 - Rewards are calculated on a daily basis, based on the amount locked and the remaining locking duration. Concretely, the weight for a locked amount is given by `lockedAmount * (remainingLockingDurationInDays + 150)`. The total daily amount of staking rewards is then shared by all users proportional to their weight.
-- Users can claim their rewards (and restake them immediately if they want) at any time.   
+- Users can claim their rewards (and restake them immediately if they want) at any time.
+
+## Pausing a Locking Position
+
+As mentioned in several subsection before, it is possible *to pause* the countdown of a locking position. This can be done at any time. Once paused, the remaining locking duration does not decrease anymore. The owner of the locking position can *resume* the countdown of the locking period an any point in time. Once resumed, the remaining locking duration descreases again daily.
+
+**Example**: A users creates a locking position on day 1 and with a locking duration of 30 days. On day 6, the remaining locking duration is 25 days. On the same day, the user pauses the locking pausition. On day 50, the remaining locking duration is still 25. The users resumes the countdown of the locking position on this day. On day 75, the reamining locking duration is zero and the user can unlock their tokens.
+
+What are the consequences of pausing the countdown:
+
+- The voting power is boosted, i.e. increased from `lockedAmount` to `lockedAmount * (1 + remainingLockingDurationInDays/365)`.
+- The rewards paid out per day for the locking position do not decrease daily (assuming a constant total weight in the rewards contract). In contrast, the rewards paid out per day for an unpaused locking position decrease day by day.
```

Comment:
Here we could be a bit more precise to avoid confusion. Suggestion (but again, no strong opinion):

```suggestion
- The weight on the rewards calculation (`lockedAmount * (remainingLockingDurationInDays + 150)` does not decrease daily (as for unpaused locking position) but remains fixed for all the duration that the position remains paused. This results to higher staking rewards.
```

Reply:
Your suggestion is more precise, but I nevertheless prefer the current way as the main point is easier to grasp in my opinion: higher rewards.

Reply:
Ok. The main point is that in my formulation we don't need to make any assumptions on the total weight. On the other hand, the statement that by default rewards decrease daily is true only under this assumption. This is why I suggested this.

Reply:
ok, I took you suggestion with small modifications :)